ingest-app/src/cmr/ingest/api/bulk.clj

```diff
@@ -12,7 +13,22 @@
    [cmr.ingest.config :as ingest-config]
    [cmr.ingest.data.bulk-update :as data-bulk-update]
    [cmr.ingest.services.bulk-update-service :as bulk-update]
-   [cmr.ingest.services.ingest-service :as ingest]))
+   [cmr.ingest.services.ingest-service :as ingest]
+   [cmr.transmit.echo.tokens :as tokens]))
+
+(def user-id-cache-key
```

Comment:
This one is used outside of bulk ingest - it should stay in core.

Reply:
Oh, I just made a copy, I forgot bulk.clj is in api and can reference the api core :)  Done.

Reply:
Done. Removed user-id-cache-key and moved get-user-id to api core.  Refactored the set-user-id there.